go/stellar/build.go

```diff
@@ -185,6 +287,16 @@ func BuildPaymentLocal(mctx libkb.MetaContext, arg stellar1.BuildPaymentLocalArg
 					})
 				}
 			}
+			offerAdvancedForm, err := ShowAdvancedSendBanner(mctx, remoter, arg.From, arg.To)
```

Comment:
This is a good candidate for `buildPaymentCache`. This function is called roughly every keystroke when composing a send. And `ShowAdvancedSendBanner` looks to be hitting a remote every time. Could cache in `bpc` ala `LookupRecipient`. Another strategy could be to only recalculate if `From` or `To` change, though there's nothing like that right now.